devtools/conda-envs/test_env.yaml

```diff
@@ -50,13 +49,3 @@ dependencies:
   - nbval
   # No idea why this is necessary, see https://github.com/openforcefield/openff-toolkit/pull/1821
   - nomkl
-  - mypy
```

Comment:
(not blocking) just double checking that this removal is intended

Reply:
Yep - it's not run when this environment file is used